src/core/inlines.js

```diff
@@ -145,9 +145,10 @@ function inlineXrefMatches(matched, text) {
   }
 
   const node = idlStringToHtml(ref);
-  // If it's inside a dfn, it should just be coded, not linked.
+  // If it's inside a dfn or a `a`, it should just be coded, not linked.
   // This is because dfn elements are treated as links by ReSpec via role=link.
-  const renderAsCode = !!text.parentElement.closest("dfn");
+  const renderAsCode =
+    !!text.parentElement.closest("dfn") || !!text.parentElement.closest("a");
```

Comment:
I suggest to merge the selectors:

```suggestion
    !!text.parentElement.closest("dfn,a");
```